sci-mathematics/cgal/cgal-5.6.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Modules provided by dev-cpp/eigen
+CMAKE_REMOVE_MODULES_LIST=( FindEigen3 )
+inherit cmake
+
+MY_P=CGAL-${PV}
+DESCRIPTION="C++ library for geometric algorithms and data structures"
+HOMEPAGE="https://www.cgal.org/"
+SRC_URI="
+	https://github.com/CGAL/cgal/releases/download/v${PV}/${MY_P}.tar.xz
+	doc? ( https://github.com/CGAL/cgal/releases/download/v${PV}/${MY_P}-doc_html.tar.xz )"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="LGPL-3 GPL-3 Boost-1.0"
+SLOT="0/14"
+KEYWORDS="amd64 ~arm64 ~ppc64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="doc examples"
```

Comment:
checking the keywords, why straight to stable for amd64? also new ppc64, tested?